docs/content/manual/release-specific/v1.4.0/test-online-expansion.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Test Online Expansion
+---
+
+## Related issue
+https://github.com/longhorn/longhorn/issues/1674
+
+## Test online expansion with continuous reading/writing
+
+**Given** Prepare a relatively large file (5Gi for example) with the checksum calculated.
+
+*And* Create and attach a volume.
+
+*And* Monitor the instance manager pod logs.
+
+**When** Use `dd` to copy data from the file to the Longhorn block device.
+
+```
+dd if=/mnt/data of=/dev/longhorn/vol bs=1M
+```
+
+*And* Do online expansion for the volume during the copying.
+
+**Then** The expansion should success.
+
+*And* Should not see any io or tgt related error logs in the instance manager pods.
+
+**When** Use `dd` to calculate the data checksum for the Longhorn block device..
+
+```
+dd if=/dev/longhorn/vol bs=1M count=5000 status=none | md5sum
+```
+
+*And* Do online expansion for the volume during the calculation.
+
+**Then** The expansion should success.
```

Comment:
Can we elaborate this point as something like this `The expansion should succeed and the size of the volume needs to be verified on the attached node.`

Reply:
Sure. Let me add one more statement: `The corresponding block device on the attached node is expanded.`